spec/magma_attribute_spec.rb

```diff
@@ -183,9 +183,9 @@
       attribute = Magma::Attribute.new("name", model, {
         validation: { type: "Regexp", value: /^[a-zA-Z]{1}$/ }
       })
-      json_validation_object = attribute.json_template[:validation_object].as_json
+      json_validation_object = attribute.json_template[:validation]
 
-      expect(json_validation_object).to eq("{\"type\":\"Regexp\",\"value\":\"(?-mix:^[a-zA-Z]{1}$)\"}")
+      expect(json_validation_object).to be_a(Magma::RegexpValidationObject)
```

Comment:
This spec is in the `describe "#validation_object"` block, but it should be in the `describe "#json_template"` block at the beginning of the file.

Now that the `validation_object` is included as a hash in the `json_template`, this spec needs to be updated.

```
  1) Magma::Attribute#validation_object contains a ValidationOjbect
     Failure/Error: expect(json_validation_object).to be_a(Magma::RegexpValidationObject)
       expected {:type=>"Regexp", :value=>/^[a-zA-Z]{1}$/} to be a kind of Magma::RegexpValidationObject
     # ./spec/magma_attribute_spec.rb:188:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:185:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:184:in `block (2 levels) in <top (required)>'
```